playground/hmr/__tests__/hmr.spec.ts

```diff
@@ -877,7 +877,25 @@ if (import.meta.hot) {
     editFile('self-accept-within-circular/c.js', (code) =>
       code.replace(`export const c = 'c'`, `export const c = 'cc'`),
     )
-    await untilUpdated(() => el.textContent(), 'cc')
+    await untilUpdated(
+      () => page.textContent('.self-accept-within-circular'),
+      'cc',
+    )
```

Comment:
```suggestion
    await untilUpdated(() => el.textContent(), 'cc')
```
I guess this diff is not necessary.

Reply:
This was needed since it now triggers a page reload for correctness, and the `el` reference is lost.

Reply:
That makes sense 👍